tap_google_ads/streams.py

```diff
@@ -307,8 +310,8 @@ def sync(self, sdk_client, customer, stream, config, state): # pylint: disable=u
 
                 singer.write_record(stream_name, record)
 
-        state = singer.bookmarks.set_currently_syncing(state, None)
-
+        state = singer.bookmarks.set_currently_syncing(state, (None, None))
```

Comment:
Should this just be `None`? Is there a world when we'd want it to be like `["campaigns",None]`?

Reply:
After thinking through your other comment, I think there is never value in `["campaigns", None]`.  It would be a valid state you can start the tap with, but I like our new approach of letting the new state overwrite the old one